ticker/src/main/java/com/robinhood/ticker/TickerColumnManager.java

```diff
@@ -36,33 +39,48 @@
     final ArrayList<TickerColumn> tickerColumns = new ArrayList<>();
     private final TickerDrawMetrics metrics;
 
-    // The character list that dictates how to transition from one character to another.
-    private char[] characterList;
+    private List<char[]> characterLists;
```

Comment:
just a thought - would it be painful to make the character lists just a string? string has `.charAt(int index)`, and then we could avoid needing all the char arrays everywhere

Reply:
good thought, I can change the public API to use a string for ease of use, but internal states should be kept at char array because that's the form we need to do the actual drawing on the canvas.

Reply:
I'm not sure if String -> char[] is a "free" conversion. If not, I'd just keep the char[] the whole way through.